tools/ddrgen/src/scanners/dwarf/AixDwarfParser.cpp

```diff
@@ -705,18 +279,15 @@ parseDeclarationLine (const string data, Parsed_Declaration_Data *parsedData)
 static int
 parseDwarfDie(const string line)
 {
-	
 	int ret = DW_DLV_OK;
 	/* get the type of the data of the line being parsed */
 	Parsed_Declaration_Data *parsedDeclarationData = new Parsed_Declaration_Data();
```

Comment:
It seems you updated it without following my suggestion. The variable parsedDeclarationData should not be a pointer. Please see the code example in my previous comment.